### DISCUSSION TITLE
Import error when running quickstart in pytorch-latest container

### DISCUSSION THREAD

**Discussion by NoaMills-USDA** [azavea/raster-vision#1600] [Q&A] [answered]
I pulled the latest pytorch image on 12/9/22 (which is still the latest image to date), but I get an error running the quickstart in this container. I can share the whole traceback if needed, but this last section may be sufficient.

`    from torch.hub import _import_module
ImportError: cannot import name '_import_module' from 'torch.hub' (/home/noa.mills/.local/lib/python3.9/site-packages/torch/hub.py)
`
Any ideas why I'm getting this error? I tried importing it on my own, and it failed. I can't seem to find any information for how to install or import it properly.

**Comment by AdeelH** [comment, marked as answer]
>`/home/noa.mills/.local/lib/python3.9/site-packages/torch/hub.py`

That doesn't look like a path inside the docker image. Are you sure you're running it inside the container?

That file should be located at `/opt/conda/lib/python3.9/site-packages/torch/hub.py` inside the container.

**Reply by NoaMills-USDA** [reply]
Thanks for the tip. I was able to figure it out. I converted the docker image into a singularity image, and singularity automatically binds your home directory and working directory. I happened to have a broken version of pytorch in my home directory, and it looked there first for torch/hub.py. Once I deleted pytorch from my home directory, I stopped getting this error.
